regalii/regaliator
# 1.5

#### What's this PR do?
Refactor gem to only support v1.5

#### Where should the reviewer start?
Regaliator gem

#### How should this be manually tested?
  1. Create a regaliator client with a valid configuration
  2. Use the methods and everything should work as expected

#### Any background context you want to provide?
Implementing the same versioning system as `rails` (branching)

#### Questions:
- Question 1
